compile warnings after <string.h> include removed

After removal of `string.h` include (https://github.com/bitcoin-core/secp256k1/commit/a6c6f994b55619154fda3de20527dc560b8ca1ed):

```
In file included from src/tests.c:16:
In file included from ./src/secp256k1.c:14:
./src/ecmult_impl.h:186:9: warning: implicitly declaring library function
      'memcpy' with type 'void *(void *, const void *, unsigned long)'
        memcpy(dst->pre_g, src->pre_g, size);
        ^
./src/ecmult_impl.h:186:9: note: include the header <string.h> or explicitly
      provide a declaration for 'memcpy'
./src/ecmult_impl.h:230:5: warning: implicitly declaring library function
      'memset' with type 'void *(void *, int, unsigned long)'
    memset(wnaf, 0, len * sizeof(wnaf[0]));
    ^
./src/ecmult_impl.h:230:5: note: include the header <string.h> or explicitly
      provide a declaration for 'memset'
2 warnings generated.
```

gcc-4.8.3 on openSUSE:

```
In file included from src/secp256k1.c:14:0:
src/ecmult_impl.h: In function 'secp256k1_ecmult_context_clone':
src/ecmult_impl.h:186:9: warning: implicit declaration of function 'memcpy' [-Wimplicit-function-declaration]
         memcpy(dst->pre_g, src->pre_g, size);
         ^
src/ecmult_impl.h:186:9: warning: incompatible implicit declaration of built-in function 'memcpy' [enabled by default]
src/ecmult_impl.h: In function 'secp256k1_ecmult_wnaf':
src/ecmult_impl.h:230:5: warning: implicit declaration of function 'memset' [-Wimplicit-function-declaration]
     memset(wnaf, 0, len * sizeof(wnaf[0]));
     ^
src/ecmult_impl.h:230:5: warning: incompatible implicit declaration of built-in function 'memset' [enabled by default]
```

All logs here are from standalone compile.

--- a/src/ecmult_impl.h
+++ b/src/ecmult_impl.h
@@ -11,6 +11,8 @@
 #include "scalar.h"
 #include "ecmult.h"
 
+#include <string.h>
+
 /* optimal for 128-bit and 256-bit exponents. */
 #define WINDOW_A 5
 

